--- /tmp/phenny/irc.py	2010-01-30 12:23:24.000000000 +0100
+++ irc.py	2010-02-28 23:14:31.000000000 +0100
@@ -71,7 +71,7 @@
       if self.verbose: 
          message = 'Connecting to %s:%s...' % (host, port)
          print >> sys.stderr, message,
-      self.create_socket(socket.AF_INET, socket.SOCK_STREAM)
+      self.create_socket(socket.AF_INET6, socket.SOCK_STREAM)
       self.connect((host, port))
       try: asyncore.loop()
       except KeyboardInterrupt: 
